### PR TITLE
Interactives UX: Finalize insert code

### PR DESCRIPTION
**Pull request by jmif**
Finalize insert code UX

### CODE REVIEW

File: ozaria/site/components/interactive/PageInteractive/insertCode/index.vue [hunk 1]
```diff
@@ -187,7 +188,24 @@
...
+      window.removeEventListener('resize', this.onResize)
+    },
+
     methods: {
+      onResize () {
+        if (!this.codemirrorReady) {
+          return
+        }
+
+        window.requestAnimationFrame(() => this.codemirror.setSize('100%', '100%'))
```
Comment (AndrewJakubowicz) [inline ozaria/site/components/interactive/PageInteractive/insertCode/index.vue]: This seems like a way to defer the `setSize` method call. I'm interested in what the pros are of this approach.

Comment (jmif) [reply ozaria/site/components/interactive/PageInteractive/insertCode/index.vue]: Yup, this queues the changes until right before the browsers next render.  It's a performance optimization (often used in animations) so allow visual updates to only run on browser repaints.  This is usually synced with the browser frame rate.

**Comment by jmif** [comment]
@AndrewJakubowicz good catch, this was not handled in another PR.  Fixed.

File: ozaria/site/components/interactive/PageInteractive/insertCode/index.vue [hunk 2]
```diff
@@ -211,13 +236,17 @@
         }
 
         const lineToReplace = this.localizedInteractiveConfig.lineToReplace - 1
+        console.log(lineToReplace)
```
Comment (AndrewJakubowicz) [inline ozaria/site/components/interactive/PageInteractive/insertCode/index.vue]: Nitpick: Left in from debugging?

Comment (jmif) [reply ozaria/site/components/interactive/PageInteractive/insertCode/index.vue]: Yep removed

**Pull request merged**